sources/postgres/README.md

```diff
@@ -24,41 +24,44 @@ on a PostgreSQL database (via go's database/sql package).
 To use HarbourBridge on a PostgreSQL database called mydb using pg_dump output,run:
 
 ```sh
```

Comment:
Same comment as for mysql. I think we need to introduce the subcommands at some point.